pkg/ocm/helpers.go

```diff
@@ -98,15 +103,32 @@ func ClusterNameValidator(name interface{}) error {
 	if str, ok := name.(string); ok {
 		str := strings.Trim(str, " \t")
 		if !IsValidClusterName(str) {
-			return fmt.Errorf("Cluster name must consist of no more than 15 lowercase " +
-				"alphanumeric characters or '-', start with a letter, and end with an " +
-				"alphanumeric character.")
+			return fmt.Errorf("Cluster name must consist of no more than %d lowercase "+
+				"alphanumeric characters or '-', start with a letter, and end with an "+
+				"alphanumeric character.", MaxClusterNameLength)
 		}
 		return nil
 	}
 	return fmt.Errorf("can only validate strings, got '%v'", name)
 }
 
+func IsValidClusterDomainPrefix(domainPrefix string) bool {
+	return clusterDomainPrefixRE.MatchString(domainPrefix)
+}
+
+func ClusterDomainPrefixValidator(domainPrefix interface{}) error {
```

Comment:
@machi1990 Could we have a unit test for this as well please.

cheers!

Reply:
done